src/inc/msquic.h

```diff
@@ -894,6 +894,7 @@ typedef struct QUIC_SCHANNEL_CREDENTIAL_ATTRIBUTE_W {
 #ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
 #define QUIC_PARAM_LISTENER_CIBIR_ID                    0x04000002  // uint8_t[] {offset, id[]}
 #endif
+#define QUIC_PARAM_DOS_MITIGATION                       0x04000004
```

Comment:
```suggestion
#define QUIC_PARAM_DOS_MODE_EVENTS                       0x04000004  // BOOLEAN
```
Instead of calling it DOS_MITIGATION, let's call it DOS_MODE and add EVENTS to indicate this is for opting into receiving events for a change in DOS_MODE.

Reply:
Done.